fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/digitalcash/HistoryListAdapter.kt

```diff
@@ -51,7 +51,10 @@ class HistoryListAdapter(
     holder.transactionIdValue.text = transactionId
     holder.transactionProvenanceTitle.setText(
         if (element.isSender) R.string.digital_cash_to else R.string.digital_cash_from)
-    holder.transactionProvenanceValue.text = element.senderOrReceiver
+    holder.transactionProvenanceValue.text =
+        if (element.senderOrReceiver.encoded == viewModel.organizer.encoded)
+            element.senderOrReceiver.encoded + " (organizer)"
```

Comment:
Shouldn't this be
`element.senderOrReceiver.getUsername()+ " (organizer)"`

Reply:
This is the case I explained in the PR description, where the Organizer key (used when Issuing) is longer than allowed for a `PublicKey`. Therefore we cannot use the `getUsername `function.

> For organizer transactions, the full public key hash is displayed with "(organizer)" appended, as the public key is longer than 32 bits and cannot be instantiated as PublicKey. Also, it juste made no sense to label this public key under a username anyway.